mdtools/__init__.py

```diff
@@ -49,17 +49,39 @@
     __copyright__,
     __copyright_notice__,
 )
-import mdtools.box
-import mdtools.check
-import mdtools.dtrj
+from mdtools import box
+from mdtools import check
+from mdtools import dtrj
 import mdtools.dynamics as dyn
```

Comment:
Shouldn't these imports then also look like `from mdtools import dynamics as dyn`?

Reply:
They don't have to be because flake8 recognizes those correctly. The `as` seems to make a difference. Which way do you prefer? I can change these to make it consistent.

Reply:
I would prefer to have it consistent, i.e. to use the `from mdtools import ... [as ...]` syntax for all imports in the `__init__.py`, if you don't mind changing it.

Reply:
Sure enough, done.